### PR TITLE
Create ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafkk

Ticket: [DM-3868](https://cumulocity.atlassian.net/browse/DM-3868)
Original PR: [7226](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7226)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed issues in the asset properties widget on device info page.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md]: The Claude AI suggested to use the following title:
```suggestion
title: Asset properties widget on device info page fixed
```

File: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3868
+version: 1020.28.12
+---
+Fixed issues in the asset properties widget on device info page.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md]: The Claude AI suggested to use the following body:
```suggestion
The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. Users relying on the asset properties widget for device information can now be assured the data shown is accurate and complete.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@dafkk Please check this PR.

File: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md]:
```suggestion
The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. The updated asset properties widget in the device info page now contains 5 default properties - ID, name, type, last updated and creation time.
```

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3868
+version: 1020.28.12
+---
+The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. The updated asset properties widget in the device info page now contains 5 default properties - ID, name, type, last updated and creation time.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md]:
```suggestion
The "Asset properties" widget on the **Info** tab in the device details was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the "Asset properties" widget now correctly displays all relevant properties for a device asset. It contains 5 default properties - ID, name, type, last updated and creation time.
```

File: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Asset properties widget on device info page fixed
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md]:
```suggestion
title: Asset properties widget in device details correctly displays all relevant properties



```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**